### DISCUSSION TITLE
How to generate XML Report from cypress-allure-plugin

### DISCUSSION THREAD

**Discussion by Animeshkr29** [cypress-io/cypress#16909] [Plugins]
Hi Developers Team 

i want to integrate test rail with Allure Cypress report , can any one please help to know how to get XML Report if i am using Allure Report Plugin in Cypress .

**Comment by bahmutov** [comment]
I think this question should be directed at the allure plugin rather than at cypress repo

**Reply by Animeshkr29** [reply]
Thanks for Quick reply , Will it possible to help me how i can contact them via Git hub ?

**Comment by bahmutov** [comment]
That I’m not sure. Try finding their GitHub repo or contact information

**Reply by Animeshkr29** [reply]
Thanks , i find it here 
https://github.com/allure-framework


Closing this Discussion
